wa/instruments/perf.py

```diff
@@ -77,6 +77,17 @@ class PerfInstrument(Instrument):
                   description="""Provides labels for pref output. If specified, the number of
                   labels must match the number of ``optionstring``\ s.
                   """),
+        Parameter('perf_mode', kind=str, default='stat', allowed_values=['stat', 'record'],
```

Comment:
Parameter can be renamed to just `"mode"`, given that it is for `perf` instrument, calling it `perf_mode` is redundant.

Reply:
Thanks, changed in latest version.